client/ClickWrapper.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const ClickWrapper = (ComponentToWrap, moduleName) => {
+  return (props) => (
+    <div onClick={(e) => {
+      console.log('clicked element:', e.target.outerHTML, '\nclicked module', moduleName, '\ntimestamp:', new Date);
```

Comment:
No rush but we should remove this console.log in a future PR!

Reply:
Totes, will do.